README.rst

```diff
@@ -11,29 +11,27 @@ EModelRunner is a python library designed to run the cell models provided by the
 
 It consists of two main modules:
 
-  - one module designed to show the synapse plasticity phenomenon. It has the following features:
-    
-    - ability to run post-synaptic cell with pre-recorded output of pre-synaptic cell
-    - ability to run 'true' pair simulations, where both the pre-synaptic and the post-synaptic cells are run at the same time
-    - analysis tool to compute the change of EPSPs
-
-  - one module designed to run the cells from the SomatoSensoryCortex portal, with the following features:
-    
-    - ability to use Steps and Ramps protocols
-    - synapse stimulation
-    - ability to produce hoc files to run the cells
-    - a GUI
+- one module designed to show the synapse plasticity phenomenon. It has the following features:
+
+  - ability to run post-synaptic cell with pre-recorded output of pre-synaptic cell
+  - ability to run 'true' pair simulations, where both the pre-synaptic and the post-synaptic cells are run at the same time
+  - analysis tool to compute the change of EPSPs
+
+- one module designed to run the cells from the SomatoSensoryCortex portal, with the following features:
+
+  - ability to use Steps and Ramps protocols
+  - synapse stimulation
+  - ability to produce hoc files to run the cells
+  - a GUI
 
 Note that EModelRunner is a convenience tool and does not aim to be a universal simulator.
 
 Installing EModelRunner
 =======================
 
-The usual way to install EModelRunner is using pip.
-
-In that case, you probably want to use a python virtual environment.
+The usual way to install EModelRunner is using pip. In that case, you probably want to use a python virtual environment.
 
-Pip install emodelrunner from PyPi will be available after open sourcing::
+Instll uing ``pip``::
```

Comment:
Also a small type here

Reply:
thanks for noticing, should be fixed now